src/exercism.ts

```diff
@@ -0,0 +1,173 @@
+const helpOption: Fig.Option[] = [
+  {
+    name: ["-h", "--help"],
+    description: "More information about command",
+  },
+];
```

Comment:
I saw that this `helpOption` is included everywhere -- it's a nice touch of polish, but you can get the same behavior with less copy and pasting by setting [`isPersistent`](https://fig.io/docs/reference/option#ispersistent) to true!

Just make sure `helpOption` added to the root subcommand and the `--help` flag will appear for all child subcommands as well. 

```suggestion
const helpOption: Fig.Option[] = [
  {
    name: ["-h", "--help"],
    description: "More information about command",
    isPersistent: true,
  },
];
```

Reply:
Thanks a lot for the feedback @mschrage! Using `isPersistent` is indeed a nice touch that makes the whole thing a bit clearer. I removed the `helpOption` because the same option had already been available at the top level.

This also made me notice that the global flags were not yet available for all subcommands as intended.